src/test/apollo-server/utils.ts

```
@@ -0,0 +1,110 @@
+    }
+
+    return await response.text();
+  } catch (error) {
+    if ((error as Error).name === 'AbortError') {
+      throw new Error('Schema fetch timeout after 5 seconds');
+    }
+    throw new Error(`Schema fetch failed: ${(error as Error).message}`);
+  }
+}
+
+let userCounter: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)